python/rmm/rmm/allocators/cupy.py

```diff
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from rmm import _lib as librmm
+from rmm import python as librmm
```

Comment:
It's weird to me that python is imported as `librmm` and `cpp` is imported as `lib`. RMM C++ is referred to as `librmm` elsewhere.

Suggest either `from rmm import python as rmm` or take a cue from cuDF and 

```
from rmm import python as pylibrmm
from rmm import cpp as librmm
```

You may even rename the directories to match, although "python" and "cpp" is pretty clear.

Reply:
Okay I applied your suggestion. Can I get your thoughts @wence-?

Reply:
Ah well it looks like Mark had [the same idea as me](https://github.com/rapidsai/rmm/pull/1676#discussion_r1769339111). I would suggest that we do rename the directories rather than needing to alias the imports everywhere, and also to make it easier for RAPIDS developers switching between projects who might expect a specific structure.

Reply:
So you want to rename python/ --> pylibrmm/ and cpp/ --> librmm/?

Reply:
Yup.